picohttp/democlient.c

```diff
@@ -680,7 +680,7 @@ char const * demo_client_parse_stream_number(char const * text, uint64_t default
         *number = 0;
         do {
             *number *= 10;
-            *number += *text++ - '0';
+            *number += (uint64_t)(*text++) - (uint64_t)'0';
```

Comment:
I'd leave that as it was. This operation is safe to be performed on the char type and then converted to uint64_t. Casting it makes it just more unreadable :)

Reply:
I rewrote it for clarity